rbi/tools/sync-rdoc.rb

```diff
@@ -583,6 +583,12 @@ def run!(argv)
       return -1
     end
 
+    if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2.7')
+      puts "Warning! Detected RUBY_VERSION=#{RUBY_VERSION}."
+      puts "This script uses the currenet Ruby version to parse documentation from."
```

Comment:
Typo (`currenet`).

(GitHub won't let me make a "suggestion" here for some reason)